phpBB/includes/mcp/mcp_post.php

```diff
@@ -549,27 +549,27 @@ function change_poster(&$post_info, $userdata)
 	$from_username = $post_info['username'];
 	$to_username = $userdata['username'];
 
-	// Renew post info
-	$post_info = phpbb_get_post_data(array($post_id), false, true);
-
-	if (!sizeof($post_info))
-	{
-		trigger_error('POST_NOT_EXIST');
-	}
-
-	$post_info = $post_info[$post_id];
-
 	/**
 	* This event allows you to perform additional tasks after changing a post's poster
 	*
 	* @event core.mcp_change_poster_after
 	* @var	array	userdata	Information on a post's new poster
 	* @var	array	post_info	Information on the affected post
-	* @since 3.1.6-RC1
+	* @since 3.1.7-RC1
```

Comment:
You shouldn't change @since but add a @change tag, example: https://github.com/phpbb/phpbb/pull/3946/files#diff-a661e9c889d836377e0341f8e90befacR4288